src/main/java/com/example/demo/service/AmazonS3Service.java

```diff
@@ -0,0 +1,11 @@
+package com.example.demo.service;
+
+import com.amazonaws.services.s3.model.S3Object;
+
+import java.io.File;
+
+public interface AmazonS3Service {
+    void save(File file);
+
+    S3Object getFile(String fileName);
```

Comment:
Can't see the file in returning. I guess the better name is getS3Object or just get